### DISCUSSION TITLE
What NTP server is good for Chia-blockchain?

### DISCUSSION THREAD

**Discussion by grafbcn** [Chia-Network/chia-blockchain#5496] [Support]
Hi.

I think Microsoft NTP server is not best option and I have many many problems with it. What you think is the correct NTP server to sync my server?
We need to use NTP protocolo or SNTP is allowed option?

**Comment by firmicus** [comment]
https://www.ntppool.org/

**Reply by grafbcn** [reply]
Is official used NTP pool for chia? I undestand in the world we have many very good NTP pools, but I want the same if is possible.

**Comment by BrandtH22** [comment]
@grafbcn , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
